src/mds/snap.cc

```diff
@@ -146,6 +146,8 @@ void sr_t::encode(bufferlist& bl) const
   encode(past_parents, bl);
   encode(past_parent_snaps, bl);
   encode(flags, bl);
+  encode(last_modified, bl);
+  encode(change_attr, bl);
```

Comment:
As we discussed and debugged today. This will make the kclient and old user space clients decode it as corrupted snap trace.

So we need to make it to be compitable with old clients.